conferences/test_helpers.py

```diff
@@ -47,12 +47,15 @@ def new_zosia(commit=True, **kwargs):
 USER_DATA = [
     ['john', 'lennon@thebeatles.com', 'johnpassword'],
     ['ringo', 'starr@thebeatles.com', 'ringopassword']
-
 ]
 
 
-def new_user(ind):
-    return User.objects.create_user(*USER_DATA[ind])
+def new_user(ind, **kwargs):
+    user = User.objects.create_user(*USER_DATA[ind])
```

Comment:
You [can pass extra fields](https://github.com/django/django/blob/master/django/contrib/auth/models.py#L151) in create user method :)